### PR TITLE
Fix #36848 24xxx.me

### LINKED ISSUE #36848: 24xxx.me
### Issue URL (Ads)
[http://www.24xxx.me/video/chuvak-poimel-i-konchil-v-45890.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fwww.24xxx.me%2Fvideo%2Fchuvak-poimel-i-konchil-v-45890.html)
### Comment
Username: @spok1
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.2.1
Browser: | Chrome
Stealth mode options: | Hide your search queries, Remove X-Client-Data header from HTTP requests, Strip URLs from tracking parameters, Hide your Referrer from third-parties
Filters: | AdGuard Russian,<br>AdGuard Annoyances,<br>AdGuard Base,<br>AdGuard Tracking Protection,<br>AdGuard Social Media

### PR DESCRIPTION
**Pull request by tvinzz**
https://github.com/AdguardTeam/AdguardFilters/issues/36848

### CODE REVIEW

**Review by slavaleleka — CHANGES_REQUESTED** [review]
I guess in this case we can add 
`||cdn-*.cdnbmb.com^$domain=24xxx.me`

**Comment by Alex-302** [comment]
and block VAST source - `u2bmco.com`

**Review by Alex-302 — CHANGES_REQUESTED** [review]
and block VAST source - `u2bmco.com`

**Comment by tvinzz** [comment]
changed the first rule and blocked VAST

File: RussianFilter/sections/specific.txt [hunk 1]
```diff
@@ -36,6 +36,8 @@ oka.fm##.rek-module
 pickpoint.ru##.bn-index-ad1
 mangalib.me##.ads
 gorlonos.com###DDWidgetSearch
+||cdn-*.cdnbmb.com^$domain=24xxx.me
+||u2bmco.com^$domain=24xxx.me
```
Comment (Alex-302) [inline RussianFilter/sections/specific.txt]: this is ad server

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: RussianFilter/sections/specific.txt (hunk 1 above)
Comment (tvinzz) [reply RussianFilter/sections/specific.txt]: moved u2bmco.com to adservers

**Review by AdamWr — APPROVED** [review]

**Review by slavaleleka — APPROVED** [review]

**Pull request merged**